### DISCUSSION TITLE
Are there any issues with changing postgres version in docker-compose?

### DISCUSSION THREAD

**Discussion by 1mike12** [amplication/amplication#4173] [Q&A]
The default docker-compose.yml sets postgres:12 as the image, but I need some other extensions (postgis and zombo), so I have a custom image. But I targeted my image to pg14. 

The build seems to have worked fine at first glance, but would there be any issues going forward?

Also, is there a dependency on the full image and not a trimmed down alpine version?

**Comment by yuval-hazaz** [comment]
@arielweinberger @abrl91 
what do you think about this?

**Comment by abrl91** [comment]
I don't think it can cause issues. In fact, with the plugin system, we provide a way to change the `db` in the `docker-compose.yml` file  and send a different file for `docker-compose.db.yml`

see example: https://github.com/amplication/plugins/tree/master/plugins/db-postgres

@1mike12 can you paste here your docker file and I'll check the plugin with its values?
You can also create your version to db-postgres plugin. I can help you with that if you want
